src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -0,0 +1,122 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import cli_command
+from azure.cli.core._util import CLIError
+
+from azure.cli.command_modules.acr.containerregistry.models import (
+    RegistryParameters,
+    RegistryProperties,
+    StorageAccountProperties
+)
+
+from ._factory import (
+    get_registry_service_client,
+    get_storage_end_point_suffix
+)
+
+from ._arm_utils import (
+    arm_get_registries_in_subscription,
+    arm_get_registries_in_resource_group,
+    arm_get_registry_by_name,
+    arm_deploy_template
+)
+
+from ._utils import (
+    get_registry_by_name,
+    get_resource_group_by_registry
+)
+
+from ._format import output_format
+
+def acr_list(resource_group=None):
```

Comment:
>resource_group [](start = 13, length = 14)

I'd recommend using 'resource_group_name' here as that is consistent with existing SDKs and there's already a global alias in place that gives you things like tab completion, etc.